scipy/special/tests/test_ndtr.py

```diff
@@ -1,11 +1,12 @@
 import numpy as np
-from numpy.testing import assert_equal, assert_allclose
+from numpy.testing import assert_equal, assert_allclose, assert_warns
 import scipy.special as sc
 
 
 def test_ndtr():
     assert_equal(sc.ndtr(0), 0.5)
     assert_allclose(sc.ndtr(1), 0.8413447460685429)
+    assert_warns(sc.SpecialFunctionWarning, sc.ndtr, 25)
```

Comment:
The preferred style in SciPy is:
```suggestion
    with pytest.warns(sc.SpecialFunctionWarning):
        sc.ndtr(25)
```

could you also add a short code comment to summarise why the warning should be thrown?

Reply:
actually, the code comment part above still remains